extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -749,7 +754,8 @@ interface StopEvaluationRunMessage {
 export type ToModelAlertsMessage =
   | SetModelAlertsViewStateMessage
   | SetVariantAnalysisMessage
-  | SetRepoResultsMessage;
+  | SetRepoResultsMessage
+  | SetReposResultsMessage;
```

Comment:
I think the difference between the `SetRepoResultsMessage` and `SetReposResultsMessage` is not clear from the name. The data seems to be exactly the same, so it's quite confusing (at least to me) when we have both messages and they have different functions. Perhaps a name like `ReplaceRepoResultsMessage` would be better (based on its function in the view code)? The same might be useful for the functions on the model alerts view, something like `updateRepoResult` and `replaceRepoResults`?

Reply:
Agree it's quite confusing. I think that `setReposResultsMessage` is the correct name for what I'm adding, but that the other messages (and related view functions) should be "addRepoResults" or something like that. Essentially `set` should be used when we're actually setting the whole thing, otherwise we should use some other term. 

I was hoping to tidy this up in a future PR, what do you think?

Reply:
That makes sense to me, I don't think this needs to be blocking for this PR.